WcaOnRails/spec/helpers/notifications_helper_spec.rb

```diff
@@ -79,8 +79,8 @@
       end
     end
 
-    context "when signed in as a board member" do
-      let(:board_member) { FactoryBot.create :user, :board_member, :wca_id }
+    context "when signed in as a WCAT member" do
+      let(:wcat_member) { FactoryBot.create :user, :wcat_member, :wca_id }
```

Comment:
@pedrosino You forgot to update the `board_member` variable below, which raised the error in the tests.